lib/ui/window.dart

```diff
@@ -178,6 +178,9 @@ enum AppLifecycleState {
   /// [Window.onBeginFrame] and [Window.onDrawFrame] callbacks.
   paused,
 
+  /// Deprecated
+  suspending,
```

Comment:
Can you augment these docs with something like:

```dart
/// This value is no longer used and is deprecated.
///
/// Developers referring to this enum value should ...
```

Reply:
Actually, let me just revert the entire commit